content/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent.md

```diff
@@ -19,7 +19,7 @@ If you don't want to reenter your passphrase every time you use your SSH key, yo
 {% data reusables.command_line.open_the_multi_os_terminal %}
 2. Paste the text below, substituting in your {% data variables.product.product_name %} email address.
   ```shell
-  $ ssh-keygen -t rsa -b 4096 -C "<em>your_email@example.com</em>"
+  $ ssh-keygen -t ed25519 -C "<em>your_email@example.com</em>"
   ```
```

Comment:
I've been advised that this command won't be supported for some GitHub users. I think we should give them the RSA command as a fallback. I suggest adding the following note immediately after the ed25519 command, indented as part of this step of the procedure:

<pre>
  {% note %}

  **Note:** If you are using a legacy system that does not support the Ed25519 algorithm, use:
  ```shell
   $ ssh-keygen -t rsa -b 4096 -C "<em>your_email@example.com</em>"
  ```

  {% endnote %}

</pre>

Reply:
having a fallback is a great idea ... totally support this. The goal would be not to rely on defaults. 

--timball